src/metrax/metrics.py

```diff
@@ -49,7 +49,24 @@ def _divide_no_nan(x: jax.Array, y: jax.Array) -> jax.Array:
 
 @flax.struct.dataclass
 class MSE(clu_metrics.Average):
-  """Computes the mean squared error for regression problems given `predictions` and `labels`."""
+  r"""Computes the mean squared error for regression problems given `predictions` and `labels`.
+  
```

Comment:
Nit: could you remove any leading whitespace in the docstrings (on this line and any below)?

i.e.:
<img width="175" alt="image" src="https://github.com/user-attachments/assets/bd3a3186-6f3b-4f59-9393-ef20b5f5cbea" />